src/scenes/level_scene.py

```diff
@@ -828,7 +828,7 @@ def is_tile_available(self, tile: Position) -> bool:
 
         return entity_on_tile is None
 
-    def get_entity_on_tile(self, tile: Position) -> Optional[Entity]:
+    def get_entity_on_tile(self, tile: Position) -> Optional[Entity, Destroyable]:
```

Comment:
`Destroyable` **is** an `Entity`.
Why adding it to the typing while it's already included?

Reply:
I think there was a downstream complaint by a function expecting a Destroyable. Hopefully this will be clearer when I roll back to a less ambitious PR.